RadixWallet/Core/SharedModels/P2P/Dapp/Models/Interaction/Transactions/SendTransactionRequest.swift

```diff
@@ -1,25 +1,60 @@
+// MARK: - RawTransactionManifest
+public struct RawTransactionManifest: Sendable, Hashable {
```

Comment:
I think perhaps it is a missed opportunity to name this `RawTransactionManifest` - it sounds like it is guaranteed to be a valid Manifest, but it can in fact be completely bonkers. So I prefer `UnvalidatedTransactionManifest`, or `UncheckedTransactionManifest`.